app/components/MixerVolmeter.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+
+<div class="volmeter">
+  <div class="volmeter-level" ref="level"></div>
+  <div class="volmeter-peak" ref="peak"></div>
+</div>
+
+</template>
+
+<script lang="ts" src="./MixerVolmeter.vue.ts"></script>
+
+<style lang="less" scoped>
+@import "../styles/index";
+
+.volmeter {
```

Comment:
It looks like this CSS was copied from `MixerItem.vue`.  Can we remove from there if this is its new home?

Reply:
👍 +1